options.html

```diff
@@ -1,148 +1,172 @@
+</head>
+<body>
+<div id="rikaikun_options">
+    <!--			<div id="tabs">
+                    <div id="general" class="tab selected-tab">General</div><div id="keyboard" class="tab active-tab">Keyboard</div><div id="worddict" class="tab">Word Dictionary</div><div id="kanjidict" class="tab active-tab">Kanji Dictionary</div><div id="namedict" class="tab">Names Dictionary</div><div id="clipboard" class="tab active-tab">Clipboard</div>
+                </div> -->
+    <form id="optform" name="optform">
+        <div id="options">
+            <div id="gencon" class="tabContent">
+                <h1>General</h1>
+                <p>Popup color:
+                    <select class="inputfield" id="popupcolor" name="popupcolor">
+                        <option value="blue">Blue</option>
+                        <option value="lightblue">Light Blue</option>
+                        <option value="black">Black</option>
+                        <option value="yellow">Yellow</option>
+                    </select>
+                    <br/>
+                    Default popup location:
+                    <select class="inputfield" id="popupLocation" name="popupLocation">
+                        <option value="0">Cursor</option>
+                        <option value="1">Top Left</option>
+                        <option value="2">Bottom Right</option>
+                    </select>
+                    <br/>
+                    <input type="checkbox" id="highlighttext" name="highlighttext">
+                    Highlight text
+                    <br/>
+                    <input type="checkbox" id="textboxhl" name="textboxhl">
+                    Highlight text inside of text boxes
+                    <br/>
+                    <input type="checkbox" id="onlyreading" name="onlyreading">
+                    Hide definitions and show only reading
+                    <br/>
+                    <input type="checkbox" id="minihelp" name="minihelp">
+                    Show mini help
+                    <br/>
+                    <br/>
+                    Show popup with delay:
+                    <input type="number" id="popupDelay" name="popupDelay" min="1"> milliseconds
+                </p>
+            </div>
+            <!-- <div id="keycon" class="tabcontent hide"> -->
+            <div id="keycon" class="tabContent">
+                <h1>Keyboard</h1>
+                Show popup only on pressed key:<br/>
+                <input type="radio" name="showOnKey" value="">Not used<br/>
+                <input type="radio" name="showOnKey" value="Alt">Alt<br/>
+                <input type="radio" name="showOnKey" value="Ctrl">Ctrl<br/>
+                <input type="radio" name="showOnKey" value="Alt+Ctrl">Alt+Ctrl<br/>
+                <br/>
+                <strong>Keys when popup is visible</strong><br><br>
+                <table cellspacing=5>
+                    <tr>
+                        <td>A</td>
```

Comment:
why did you add newlines to these options? It seems maybe harder to read than having key -> meaning on the same line?